ipfx/bin/run_feature_vector_extraction.py

```diff
@@ -120,6 +121,10 @@ def data_for_specimen_id(specimen_id, sweep_qc_option, data_source, ontology,
 
     # Calculate desired feature vectors
     result = {}
+
+    if data_source == "filesystem":
```

Comment:
a) is this still needed, or b) why not save it for all data_sources not just 'filesystem'?

Reply:
a) this is newly added for dealing with nwb2 files by given file paths.

I didn't quite follow your question b.